content/docs/testing-environments.md

```diff
@@ -1,58 +1,58 @@
 
-If you're writing a library that tests mostly browser-specific behavior, and requires native browser behavior like layout or real inputs, you could use a framework like [mocha.](https://mochajs.org/)
+Se você está criando uma biblioteca que testa em sua maioria comportamentos específicos de um navegador, e necessita de um comportamento nativo de um navegador como _layout_ ou _inputs_ de verdade, você pode usar um _framework_ como [mocha](https://mochajs.org/).
 
-In an environment where you _can't_ simulate a DOM (e.g. testing React Native components on Node.js), you could use [event simulation helpers](https://reactjs.org/docs/test-utils.html#simulate) to simulate interactions with elements. Alternately, you could use the `fireEvent` helper from [`@testing-library/react-native`](https://testing-library.com/docs/native-testing-library).
+Em um ambiente onde você _não pode_ simular um DOM (por exemplo, testes de componentes do React Native no Node.js), você poderia usar [funções auxiliares de simulação de eventos](https://reactjs.org/docs/test-utils.html#simulate) para simular interações com elementos. Como uma outra alternativa, você pode usar a função auxiliar`fireEvent` da [`@testing-library/react-native`](https://testing-library.com/docs/native-testing-library).
 
-Frameworks like [Cypress](https://www.cypress.io/), [puppeteer](https://github.com/GoogleChrome/puppeteer) and [webdriver](https://www.seleniumhq.org/projects/webdriver/) are useful for running [end-to-end tests](#end-to-end-tests-aka-e2e-tests).
+Frameworks como [Cypress](https://www.cypress.io/), [puppeteer](https://github.com/GoogleChrome/puppeteer) e [webdriver](https://www.seleniumhq.org/projects/webdriver/) são úteis para executar [testes end-to-end](#end-to-end-tests-aka-e2e-tests).
 
-### Mocking functions {#mocking-functions}
+### Fazendo o mock de funções {#mocking-functions}
 
-When writing tests, we'd like to mock out the parts of our code that don't have equivalents inside our testing environment (e.g. checking `navigator.onLine` status inside Node.js). Tests could also spy on some functions, and observe how other parts of the test interact with them. It is then useful to be able to selectively mock these functions with test-friendly versions.
+Ao escrever testes, nós gostaríamos de fazer o _mock_ nas partes do nosso código que não possuem um equivalente dentro do nosso ambiente de testes (por exemplo, checar o status `navigator.onLine` dentro do Node.js). Testes também podem dar _spy_ em algumas funções, e observar como outras partes do teste interagem com elas. Ou seja, a possibilidade de fazer o _mock_ de funções selecionadas por versões mais amigáveis para testes é algo bem útil.
```

Comment:
```suggestion
Ao escrever testes, nós gostaríamos de fazer o mock nas partes do nosso código que não possuem um equivalente dentro do nosso ambiente de testes (por exemplo, checar o status `navigator.onLine` dentro do Node.js). Testes também podem espiar algumas funções e observar como outras partes do teste interagem com elas. Portanto, a possibilidade de fazer o _mock_ de funções selecionadas por versões mais amigáveis para testes é algo bem útil.
```
Não acho que vc precisa colocar mock em itálico. Vc não fez isso em outras partes.
Com relação ao _Ou seja_, você não está explicando a frase anterior. Você quer uma consequência, portanto, _portanto_ fica melhor ali (pegou a jogada! hahaha).

Reply:
Faz sentido, acho que o _portanto_ é uma conjunção que cabe melhor nesse contexto!